pipestat/reports.py

```diff
@@ -368,6 +412,21 @@ def create_object_htmls(self, navbar, footer):
                         pass
                     else:
                         try:
+                            if self.portable:
+                                newimagepath = self._create_copy_for_porting(
```

Comment:
I would suggest using underscores to separate words in variable names, like `new_image_path` -- that's the style we're using elsewhere in these packages.

Reply:
Done!